test/helpers/kubectl.go

```diff
@@ -122,6 +122,8 @@ var (
 		"ipv6NativeRoutingCIDR":  IPv6NativeRoutingCIDR,
 
 		"ipam.operator.clusterPoolIPv6PodCIDRList": "fd02::/112",
+
+		"extraConfig.max-internal-timer-delay": "5s",
```

Comment:
Should we also add this to the GitHub Actions default values here:
https://github.com/cilium/cilium/blob/main/.github/actions/helm-default/action.yaml

Reply:
I think I'll file an issue with this and other followups if that's alright with you. There's a few more small iterative items we can do on top like:
- Apply this change to more than just cilium-agent (operator etc.)
- See if there are additional packages that can reuse time.
- Pick a different value than 5s

The PR is currently pulling in a lot of owners so I wanted to make sure we get together an implementation that works and transitions the codebase over, then we can iterate and tweak the details.

Reply:
Filed https://github.com/cilium/cilium/issues/28844